docker/base-builder/Dockerfile

```diff
@@ -52,4 +52,4 @@ COPY --from=base-clang /usr/local/lib/libc++*.a /usr/local/lib/
 COPY --from=base-clang /usr/local/include/ /usr/local/include
 
 # Use libc++ thoughout the build.
-ENV CXXFLAGS -stdlib=libc++
\ No newline at end of file
+ENV CXXFLAGS -stdlib=libc++ -pthread
```

Comment:
do you remember where this failed?
If not, it's fine I can look into this..

Reply:
I tested libpng and afl-clang-fast++ failed to link the harness binary due to the lack of -pthread.
The printed errors were related to functions in libc++ that use pthread.